test/parallel/test-child-process-flush-stdio.js

```diff
@@ -4,10 +4,14 @@ const common = require('../common');
 const assert = require('assert');
 
 // Windows' `echo` command is a built-in shell command and not an external
-// executable like on *nix
-const opts = { shell: common.isWindows };
-
-const p = cp.spawn('echo', [], opts);
+// executable like on *nix. The V4 API does not have the "shell" option to
+// spawn that we use in V4 and later so we can't use that here.
+var p;
```

Comment:
Can you use `let p` here so this doesn't cause conflicts when/if https://github.com/nodejs/node/pull/10685 is backported?